core/server/apps/private-blogging/lib/views/private.hbs

```diff
@@ -15,7 +15,7 @@
         <link rel="shortcut icon" href="{{asset "favicon.ico"}}">
         <meta http-equiv="cleartype" content="on">
 
-        <link rel="stylesheet" href="{{asset "ghost.css" ghost="true" minifyInProduction="true"}}" />
```

Comment:
Just thought we can make this change already in this PR. But i am fine with having a separated PR 👍